cookbooks/bcpc-hadoop/templates/default/hv_hive-site.xml.erb

```diff
@@ -63,13 +63,19 @@
   <name>datanucleus.fixedDatastore</name>
   <value>true</value>
 </property>
-
+<% if node.run_list.expand(node.chef_environment).recipes.include?("bcpc-hadoop::hive_hcatalog")  %>
+<property>
+  <name>hive.metastore.uris</name>
+  <value>thrift://<%="#{float_host(node[:fqdn])}:9083" %></value>
+  <description>IP address (or fully-qualified domain name) and port of the metastore hosts</description>
+</property>
```

Comment:
Not sure what difference this makes, currently on my bcpc-vm2 I have the following
````xml
<property>
  <name>hive.metastore.uris</name>
  <value>thrift://f-bcpc-vm2.bcpc.example.com:9083</value>
  <description>IP address (or fully-qualified domain name) and port of the metastore hosts</description>
</property>
````
Am I missing something